pkg/etc/config.go

```diff
@@ -63,3 +64,12 @@ func GetJobQueueConfig() (cfg JobQueueConfig, err error) {
 	err = env.Parse(&cfg)
 	return
 }
+
+func GetScannerMetadata() harbor.Scanner {
+	return harbor.Scanner{
+		Name:   "Trivy",
+		Vendor: "Aqua Security",
+		// This version corresponds to the version of the trivy binary that we add as a layer to the docker image.
+		Version: "0.1.6",
```

Comment:
We could use an env var that is also used to pull the correct version into the image? I can picture the binary being updated but forgetting to update this

Reply:
I agree. Build arg and fetching Trivy binary from GitHub releases pages should handle that. As it requires some additional effort not related to unit testing I'll do that in scope of #36